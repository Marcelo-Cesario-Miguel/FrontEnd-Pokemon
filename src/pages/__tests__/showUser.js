import React from "react"
import DeleteAccount from "../deleteAccount"
import { mount, configure } from "enzyme"
import toJson from "enzyme-to-json"
import Adapter from "enzyme-adapter-react-16"

configure({ adapter: new Adapter() })
it("Tem que renderizar o nome do usuário que está logado", () => {
  const props = {
    location: {
      state: {
        idUser: "5fbeafdd4419ee00170138fd",
      },
    },
  }
  const app = mount(<DeleteAccount {...props} />)
  expect(toJson(app)).toMatchSnapshot()
})
