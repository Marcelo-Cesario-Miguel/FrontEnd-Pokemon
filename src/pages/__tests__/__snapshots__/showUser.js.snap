// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the homepage for the non logged in user 1`] = `
<deleteAccount
  location={
    Object {
      "state": Object {
        "idUser": "5fbeafdd4419ee00170138fd",
      },
    }
  }
>
  <div
    className="body"
  >
    <Header
      siteTitle=""
    >
      <header>
        <div
          className="big-header"
        >
          <a>
            <img
              className="trainer-photo"
              src="test-file-stub"
            />
          </a>
          <a>
            <img
              className="logo-photo"
              src="test-file-stub"
            />
          </a>
          <a>
            <img
              className="menu"
              src="test-file-stub"
            />
          </a>
        </div>
        <div
          className="header"
        >
          <mockConstructor
            className="header-button"
            to="/mypokemons"
          >
            <a
              className="header-button"
              href="/mypokemons"
            >
              Meus Pok√©mons
            </a>
          </mockConstructor>
          <mockConstructor
            className="header-button"
            to="/pokedex"
          >
            <a
              className="header-button"
              href="/pokedex"
            >
              Pokedex
            </a>
          </mockConstructor>
          <a
            className="header-button"
          >
            Amigos
          </a>
          <mockConstructor
            className="header-button"
            to="/store"
          >
            <a
              className="header-button"
              href="/store"
            >
              Capturar
            </a>
          </mockConstructor>
          <mockConstructor
            className="header-button"
            to="/battle"
          >
            <a
              className="header-button"
              href="/battle"
            >
              Batalha
            </a>
          </mockConstructor>
          <mockConstructor
            className="evento"
            to="/dungeon"
          >
            <a
              className="evento"
              href="/dungeon"
            >
              Evento
            </a>
          </mockConstructor>
        </div>
      </header>
    </Header>
    <GetNames
      idUser="5fbeafdd4419ee00170138fd"
    >
      <div
        className="container"
      >
        <div
          className="senha"
        >
          <h1
            className="title-senha"
          >
            Excluir a Conta
          </h1>
          <label
            className="title-menor"
          >
            username
          </label>
          <input
            data-testid="username-field"
            id="senha"
            name="senha"
            onChange={[Function]}
            value=""
          />
          <label
            className="title-menor"
          >
            senha
          </label>
          <input
            className="last"
            data-testid="password-field"
            id="newSenha"
            name="newSenha"
            onChange={[Function]}
            type="password"
            value=""
          />
          <button
            className="button"
            onClick={[Function]}
          >
            deletar
          </button>
        </div>
      </div>
    </GetNames>
  </div>
</deleteAccount>
`;
